docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
In the body of the docs, it should always be `Elasticsearch` (not `elasticsearch`).

Reply:
done

Reply:
Apart from the inside code block, there are still some places not captured.

Reply:
Sorry. i was messagin around with the `git commit --amend` command which took away some of my work. 😄 
should be good now.